File: Beecrowd/Beginner/1101 - 1190/1165 - Prime Number/C/1165_PrimeNumber.c
#include <stdio.h>
#include <stdbool.h>

int main() {
    int number;
    scanf("%d", &number);

    while (number--) {
        int value;
        scanf("%d", &value);

        bool isPrime = true;

        for (int j = 2; j < value; j++) {
            if (value % j == 0) {
                isPrime = false;
                break;
            }
        }

        if (isPrime) printf("%d eh primo\n", value);
        else printf("%d nao eh primo\n", value);
    }
    
    return 0;
}

File: Beecrowd/Beginner/1101 - 1190/1165 - Prime Number/Python/1165_PrimeNumber.py
number = int(input())

for _ in range(number):
    value = int(input())
    is_prime = True

    for j in range(2, value):
        if value % j == 0:
            is_prime = False
            break

    if is_prime:
        print(f"{value} eh primo")
    else:
        print(f"{value} nao eh primo")
